src/client/actions/postUserDataAction.js

```diff
@@ -1,14 +1,25 @@
-import Axios from 'axios';
+import axios from 'axios';
 import { POST_USER_DATA } from './types';
 
-const postUserData = newUser => (dispatch) => {
-  Axios.post('http://localhost:8080/registration', newUser)
-    .then(({ data }) => {
-      dispatch({
-        type: POST_USER_DATA,
-        result: data,
-      });
+const baseUrl = 'http://localhost:8080';
+
+const postUserData = (rout, newUser, history) => async (dispatch) => {
```

Comment:
Just little mistake, I think you meant "routE"

Reply:
yep, thanks. Fixed.